### DISCUSSION TITLE
Can't see tip for small plots

### DISCUSSION THREAD

**Discussion by harrylojames** [observablehq/plot#1737] [Q&A] [answered]
Hi, 

For a small plot, such as a thin 1d bar chart, the tip pop up is mostly hidden.

![image](https://github.com/observablehq/plot/assets/51630653/cfad445c-3aba-4405-bd4d-f383727dfbe2)

https://observablehq.com/d/5f88da5ac9614540

I imagine I'm missing something but I can't find an intuitive way to make the whole tip appear without changing the dimensions of the plot. Specifically, the following should remain unchanged, the position of the x axis label, the distance from the bar chart to the legend and the thickness of the bar. 

The original plot tooltip notebook suggested using paddingTop but works as expected even without it. 

![image](https://github.com/observablehq/plot/assets/51630653/a9b9c38a-bd38-4a53-b857-bd7d0ce4021b)

https://observablehq.com/@mkfreeman/plot-tooltip#cell-126

**Comment by Fil** [comment, marked as answer]
Try setting the chart’s *style* to "overflow: visible".

**Reply by harrylojames** [reply]
Amazing thank you 🎉 this solved another problem I had been having!
